content/support-matrix/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Support Matrix
+description: "Support matrix data for SODA can be found here"
+weight: 5
+disableToc: false
+---
+
+This section provides information about the support matrix data for SODA. 
+Please find below delfin project which has different vendor storage devices supported data matrix.
```

Comment:
No need to mention delfin here.. this is common page holds index of all project support matrix. delfin is just one of them